### PR TITLE
Adding doc on how to set Azure CNI versions

### PR DESCRIPTION
**Pull request by PatrickLang**
These were added in ACS-Engine 0.25 but not documented.

@jackfrancis can you review since you touched the URLs last in #4097?

### CODE REVIEW

File: docs/clusterdefinition.md [hunk 1]
```diff
@@ -68,6 +68,9 @@ To learn more about supported orchestrators and versions, run the orchestrators
...
 | useInstanceMetadata             | no       | Use the Azure cloudprovider instance metadata service for appropriate resource discovery operations. Default is `true`                                                                                                                                                                                                                                                                                        |
 | useManagedIdentity              | no       | Includes and uses MSI identities for all interactions with the Azure Resource Manager (ARM) API. Instead of using a static service principal written to /etc/kubernetes/azure.json, Kubernetes will use a dynamic, time-limited token fetched from the MSI extension running on master and agent nodes. This support is currently alpha and requires Kubernetes v1.9.1 or newer. (boolean - default == false). When MasterProfile is using `VirtualMachineScaleSets`, this feature requires Kubernetes v1.12 or newer as we default to using user assigned identity. |
+| azureCNIVersion                 | no       | Deploy a specific Azure CNI version instead of the default. Example: "v1.0.14" |
```
Comment (jackfrancis) [inline docs/clusterdefinition.md]: This is a read-only property currently (populated during template generation), it doesn't actually operate as a user toggle to install a specific version of Azure CNI.

Comment (PatrickLang) [reply docs/clusterdefinition.md]: aha - I see that now. No error is returned but it didn't work

**Comment by PatrickLang** [comment]
/assign jackfrancis

Does this look ok now?

**Pull request merged**